app/services/doc_auth/mock/result_response.rb

```diff
@@ -71,6 +71,39 @@ def attention_with_barcode?
         parsed_alerts == [ATTENTION_WITH_BARCODE_ALERT]
       end
 
+      def self.create_image_error_response(status)
+        error = case status
+                when 438
+                  Errors::IMAGE_LOAD_FAILURE
+                when 439
+                  Errors::PIXEL_DEPTH_FAILURE
+                when 440
+                  Errors::IMAGE_SIZE_FAILURE
+                end
+        errors = { general: [error] }
+        message = [
+          'Unexpected HTTP response',
+          status,
+        ].join(' ')
+        exception = DocAuth::RequestError.new(message, status)
+        DocAuth::Response.new(
+          success: false,
+          errors: errors,
+          exception: exception,
+          extra: { vendor: 'Acuant' },
```

Comment:
Nit: since this is a mock result response, perhaps the vendor should be indicated to be a mock?

Reply:
Fixed.